src/meta_merge/core.cljc

```diff
@@ -1,11 +1,16 @@
 (ns meta-merge.core
   (:require [clojure.set :as set]))
 
+(defn- metadatable?
+  [obj]
+  #?(:clj (instance? clojure.lang.IObj obj)
+     :cljs ((some-fn coll? symbol?) obj)))
```

Comment:
ClojureScript has protocols we can use to test for metadata. We should use those, not test whether it's a `coll?` or a `symbol?`